tests/files.rs

```diff
@@ -0,0 +1,72 @@
+#![cfg_attr(
+    feature = "dev",
+    allow(dead_code, unused_variables, unused_imports, unreachable_code)
+)]
+#![cfg_attr(feature = "ci", deny(warnings))]
+#![deny(clippy::all)]
+
+#[path = "./utils.rs"]
+mod utils;
+
+use scriptkeeper::R;
+use utils::test_run;
+
+#[test]
+fn allows_to_mock_files_existence() -> R<()> {
+    test_run(
+        r"
+                |#!/usr/bin/env bash
```

Comment:
looks like the indentation of these strings got messed up in the move

Reply:
Good catch.